client/src/featureform/sqlite_metadata.py

```diff
@@ -496,9 +496,12 @@ def get_model(self, name, should_fetch_tags_properties):
         )[0]
 
     def get_provider(self, name, should_fetch_tags_properties):
-        return self.query_resource(
-            "providers", "name", name, should_fetch_tags_properties
-        )[0]
+        try:
+            return self.query_resource(
+                "providers", "name", name, should_fetch_tags_properties
+            )[0]
+        except:
+            raise ValueError(f"Provider {name} not found")
```

Comment:
nit: is this the only reason this throws an error?

Reply:
The only other time should be if it cant connect to the database. Can add another case